### DISCUSSION TITLE
Webhooks for teams discussions

### DISCUSSION THREAD

**Discussion by jackbuehner** [community/community#3871] [Discussions]
I have enabled `discussion` and `discussion_comment` as events to trigger a webhook. However, when I create a new discussion or discussion comment on a team, the webhook is not triggered. Are there plans to enable this?

**Comment by danburzo** [comment]
As of today, with the `discussion` and `discussion_comment` events enabled on a Slack webhook across the organization:

* Events are not triggered for creating / deleting discussions
* Comments on discussions appear as: 

> [Org/repo] New comment by danburzo on commit https://github.com/org/repo/discussions/id#discussioncomment-id

**Reply by jackbuehner** [reply]
This does not answer my question/feedback. I am aware that it works for discussions inside of repositories. Webhooks do not work for the discussions on a _team_.

**Reply by danburzo** [reply]
Oops, didn't mean this as an answer, just corroborating your experience. What I mean, in fact, is that discussion hooks don't particularly work on the repo level either (at least in my recent experience).

**Reply by jackbuehner** [reply]
Ah, that makes sense! There is definitely a bit more to do with discussion webbooks.
